release/pipline/dags/istio_daily_flow.py

```diff
@@ -19,7 +19,7 @@
 import istio_common_dag
 
 dag, copy_files = istio_common_dag.MakeCommonDag(
-    name='istio_daily_release', schedule_interval='15 9 * * *')
+    name='istio_daily_release', schedule_interval='15 17 * * *')
```

Comment:
that was some weird UTC / date mix is it not anymore? 8a pst is probably too late if it takes 1h+ to make let's keep it around 1am or maybe up to 4a but not later ?

Reply:
this was on request of @hklai

Reply:
@hklai and @rkpagadala  when back and forth on this question. There will always be people submitting code at midnight who want it in the build and others that want the build available first thing in the morning so there is no right answers. If we had people in New York working on istio 8 AM would be 11 AM witch is quite late. However under the current green build system(runs every two hours) you need to get code in by 11 PM for it to be in the next days daily. Best that we atlest make sure all code from day -1 is part of the build. The key is to not keep changing it or everyone will be confused.

Reply:
right now it seems it changed to noon somehow which is close to the worst possible time

let's do the builds between 1a and 4a PST

Reply:
I think the execution_time problem made it a lot more confusing, when the build cut at 1am was labeled as one day before.

With that now fixed, I am fine with sticking with 1am. However, that practical means the release candidate cut off day is on 14th 1am PST.

I will amend the release process to make it clear.

Reply:
if you prefer to make it 11:59 on the 14th or 1:00a on the 15 it's all good/fine - up to you as long as it's clear and at night

thanks!

Reply:
fun fact: we have to be careful to not pick 2a local time for instance that can get a) skipped, b) ran twice during daylight savings